### DISCUSSION TITLE
如果DNS泄露了，应该怎么修改？

### DISCUSSION THREAD

**Discussion by ksahdsambn** [jason5ng32/MyIP#40] [Q&A] [answered]
到路由器后台把国内和国外DNS全部修改成cloudflare或者google DNS吗？

如果是安卓手机用cfa代理网络，应该怎么修改DNS呢？

**Comment by Clorasun** [comment]
代理规则里面修改dns相关,或直接看代理工具是否有覆盖dns的功能进行修改

**Comment by jason5ng32** [comment, marked as answer]
使用全局的代理，不会有 DNS 泄露的问题。
如果使用规则代理，则可以考虑用尽量多的命中规则，让你常访问的 URL 先命中规则，然后触发远程 DNS 解析，从而避开最终的 Final 规则。
